### DISCUSSION TITLE
Cursor problem when using on single Mac

### DISCUSSION THREAD

**Discussion by Doug9482** [apohl79/audiogridder#925] [Help] [answered]
Just trying this on a single Mac (MacBook Pro max) and it seems to work well except for weird cursor behavior. When I move the cursor over the audiogridder plugin in Logic, it jumps to the window showing the instrument outside of Logic (which must be the audiogridder server) rather than letting me deal with the instrument in the audiogridder plugin in Logic.

I have the various permissions/access set in Settings/Privacy etc. Must be some other basic thing I haven't done right?

**Comment by apohl79** [comment, marked as answer]
You should disable screen capturing and enable local mode in the server settings.

**Comment by Doug9482** [comment]
Thank you Andres, that worked!  I’m still having trouble with it though. I’m on a 2021 MacBook Pro m1 max and Monterey…  , in Logic. the plugin kept saying it wasn’t able to connect to various instruments, I’ve ben trying mostly Kontakt. After rebooting it sometimes worked then stopped working. I switched to the 1.2 beta but found the same issue. 

Was going to go back to the earlier version snd start clean again and thought maybe I should clear out some of the installed files before doing so. Are there prefs files I should get rid of etc? 

Thanks again,

Doug
